Explorer/Assets/DCL/AvatarRendering/AvatarShape/AvatarBase.cs

```diff
@@ -19,15 +19,12 @@ public class AvatarBase : MonoBehaviour
         public void SetAsMainPlayer(bool isMainPlayer)
```

Comment:
This piece was a pure draft, this feature is the right moment to change it. I don't mind if it is done a bit later but by the end of this feature.